ironic-provisioning-log-watch.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/bash
+
+# Ramdisk logs path
+LOG_DIR="/shared/log/ironic/deploy"
+
+until  ls "${LOG_DIR}"/*.tar.gz 1> /dev/null 2>&1
+do
+    sleep 1
```

Comment:
I think you need the whole logic here, otherwise your container will stop after the first bunch of files.

Reply:
Agreed. I added an external while loop.